### ISSUE TITLE
correcting HU values

### ISSUE THREAD

**Issue by brandonjnelsonFDA** [bnel1201/PedSilicoICH#42]
![image](https://github.com/user-attachments/assets/86f0d48f-ad2d-4cfc-95f2-f2f27a84e14b)
Real patient CT | ground truth phantom | simulated CT, all shown at window level = 40, window width = 100

Following simulation, the HU values become lower than their set values, defined in the Phantom class constructor:

https://github.com/brandonjnelsonFDA/PedSilicoICH/blob/2894c23f45af64422de98aa305903e1735620ad6/src/pedsilicoICH/ground_truth_definition/phantoms.py#L74
```python
    def __init__(self, phantom_dir):
        self.phantom_dir = phantom_dir
        self.materials = {'gray matter': 45,
                          'white matter': 20,
                          'CSF': 15,
                          'skull': 1000}
```

I suspect this is because of the default cutoff values used in assigning material properties in XCIST when creating the material density maps: 

https://github.com/brandonjnelsonFDA/PedSilicoICH/blob/2894c23f45af64422de98aa305903e1735620ad6/src/pedsilicoICH/ground_truth_definition/phantoms.py#L44

Notably, these defaults were for a lung imaging task and thus are likely not appropriate for brain:

```python
        material_threshold_dict = dict(zip(
                                        ['ICRU_lung_adult_healthy',
                                         'ICRU_adipose_adult2',
                                         'ICRU_liver_adult',
                                         'water',
                                         'ICRU_skeleton_cortical_bone_adult'],
                                        [-1000, -200, 0, 100, 300]))
```
Thus I should update these materials and cutoffs to be more appropriate for brain, e.g. gray matter, white matter, CSF, bone. This should yield more accurate CT numbers.

See the list of materials: https://github.com/xcist/main/tree/master/gecatsim/material

But they also provide instructions to define your own materials if any arent included:

https://github.com/xcist/documentation/blob/master/tools/CatSim%20material%20calculator.xlsx

**Comment by brandonjnelsonFDA** [comment]
I found the main parameter of xcist controlling CT number was the mu water value set in the phantom config file (lowered from 0.2059 to 0.2): https://github.com/brandonjnelsonFDA/PedSilicoICH/blob/8ab0ed78c93f789b66af349e774dc7f875d0b1ee/src/pedsilicoICH/defaults/Recon_Default.cfg#L15

After doing this, this raised the white-gray matter HUs from being -20, 0 to about 25, 40 at 120 kVp, as shown in this example below:

These images are shown at the same window level: 40, window width 100 as the examples above and show that the HU values are visually and quantitatively closer to the real patient images

![image](https://github.com/user-attachments/assets/9cc7b169-0c0e-42de-a004-296ab6578824)
Synth made at 110 kVp, 350 mA, standard kernel
Real: 120 kVp smooth kernel

The lower kVp of this randomly sampled synth image shows that at least the HU values are higher and closer to what is expected in real patient scans, note the values are higher due to the lower relative kV. The higher noise can be explained by differences in kernel (standard vs smooth)

**Comment by brandonjnelsonFDA** [comment]
Once this commit gets merged into master, I will close this issue, we may still need to optimize the mu_water value (especially to match the desired kVp), but we at least identified that to be the source of the HU value issue and know how to adjust.

**Comment by brandonjnelsonFDA** [comment]
Closing this issue for now, if we still have HU accuracy problems with respect to CADt performance we can look into further optimizing thus mu water value to best match the values seen in the training data, but we can save that for another issue if needed cd02d0cdb63444d117730ecfbb07f4ad0bbdc916
